src/index.html

```diff
@@ -6,8 +6,45 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img src="./images/imac.jpeg"
+           class="card__img"
+           alt="imac">
```

Comment:
```suggestion
      <img 
        src="./images/imac.jpeg"
        class="card__img"
        alt="imac"
      >
```
Fix formatting. Also, alt attribute should be more descriptive
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/651d81d4-1422-4157-88e1-52518d2a02f8)